testsuite/cluster-test/src/lib.rs

```diff
@@ -20,12 +20,11 @@ pub mod suite;
 pub mod tx_emitter;
 
 pub mod util {
-    use std::time::{Duration, SystemTime};
+    use libra_time;
+    use std::time::Duration;
 
     pub fn unix_timestamp_now() -> Duration {
```

Comment:
can you instead replace all instances of `unix_timestamp_now()` with `libra_time::duration_since_epoch()`?